packages/dds/tree/src/codec/codec.ts

```diff
@@ -328,3 +328,22 @@ export function withSchemaValidation<
 		},
 	};
 }
+
+/**
+ * Versions of FLuid Framework client packages.
+ * @remarks
+ * Used to express compatibility requirements by indicating the oldest version with which compatibility must be maintained.
+ * @privateRemarks
+ * This scheme assumes a single version will always be enough to communicate compatibility.
+ * For this to work, compatibility has to be strictly increasing.
+ * If this is violated (for example a subset of incompatible features from 3.x that are not in 3.0 are back ported to 2.x),
+ * a more complex scheme may be needed to allow safely opting into incompatible features in those cases:
+ * such a system can be added if/when its needed since it will be opt in and thus non-breaking.
+ * @alpha
+ */
+export enum FluidClientVersion {
+	v2_0 = "v2_0",
```

Comment:
Any valuable documentation we can add to these flags? The answer may be no, but I always view the `(undocumented)` comment in the API reports as a small (excepting scenarios where docs should implicitly inherit from some parent but don't for tooling reasons...)

Reply:
Done.